docs/user/otp-deployment.md

```diff
@@ -1,5 +1,19 @@
 # OTP Deployment Guide
 
+In this guide:
+
+1. [Overview](#overview)
+2. [OTP Deployment Archtecture](#otp-deployment-architecture)
+3. [Performing an OTP Deployment](#performing-an-otp-deployment)
+
+## Overview
+
+This guide describes how to configure and deploy OTP servers using OTP Data Tools, and is for intermediate to advanced OTP Data Tools administrators.
+
+The deployment architecture diagram below depicts how OTP servers are managed by Data Tools and can be used with elastic load balancers. The user interface is deployed on Amazon S3 servers and optionally mirrored by CloudFront, a high-bandwidth content delivery mechanism. Data Tools prepares and sends the data bundle (set of GTFS feeds) and configuration properties to the OTP servers. OTP servers initialize and also fetch the Open Streets Map data using the osm-lib server.  Data Tools does not manage UI deployments at this time.
```

Comment:
The data bundle also includes OpenStreetMap data. DataTools makes the request to the osm-lib server and then creates a bundle of the resulting OSM and GTFS data.

Reply:
I included that and reworded previous sentence.